.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: Bug report
+name: Bug report [Deprecated]
```

Comment:
should we remove the "One-click reproducible issues" principle and reference the readme here as well? Just so we don't have it duplicated

Reply:
sure it makes sense, I will update it